sdks/python/run_pylint.sh

```diff
@@ -62,13 +62,16 @@ echo "Running pylint for module $MODULE:"
 pylint -j8 "$MODULE" --ignore-patterns="$FILES_TO_IGNORE"
 echo "Running pycodestyle for module $MODULE:"
 pycodestyle "$MODULE" --exclude="$FILES_TO_IGNORE"
+echo "Running flake8 for module $MODULE:"
+flake8 $MODULE --count --select=E999 --show-source --statistics
 echo "Running isort for module $MODULE:"
 # Skip files where isort is behaving weirdly
 ISORT_EXCLUDED=(
   "apiclient.py"
   "avroio_test.py"
   "datastore_wordcount.py"
   "datastoreio_test.py"
+  "hadoopfilesystem.py"
```

Comment:
Why do we need exceptions for a growing list of files?

+ @ehudm specifically for this file.

Reply:
Just saw this (my username is udim). Was wondering why this file was added to this list?